vumi/persist/tests/test_model.py

```diff
@@ -1197,6 +1202,11 @@ def test_reverse_foreignkey_fields(self):
         results = yield s2.backlinks.foreignkeymodels()
         self.assertEqual(sorted(results), ["bar1", "bar2"])
 
+        s2 = yield simple_model.load("foo")
+        results = yield s2.backlinks.foreignkeymodel_keys()
+        self.assertEqual(sorted(results), ["bar1", "bar2"])
+        self.assertEqual(results.has_next_page(), False)
```

Comment:
Should we maybe test that pagination is working properly here: using `max_results` to limit the returned keys, checking the returned keys, checking that `has_next_page()` is `True`, using `.next_page()`, checking the returned keys and checking that `has_next_page()` is `False`?

Or is there a reason why just a single `has_next_page()` check is sufficient?